scipy/optimize/_lsq/lsq_linear.py

```diff
@@ -54,16 +59,17 @@ def lsq_linear(A, b, bounds=(-np.inf, np.inf), method='trf', tol=1e-10,
         Method to perform minimization.
 
             * 'trf' : Trust Region Reflective algorithm adapted for a linear
-              least-squares problem. This is an interior-point-like method
-              and the required number of iterations is weakly correlated with
+              least-squares problem. This is an interior-point-like method,
+              which requires the number of iterations weakly correlated with
```

Comment:
I don't think that "to be" will fix the issue with this sentence. I changed it, please recheck.